pb/ctrl_pb/ctrl.proto

```diff
@@ -30,13 +30,19 @@ message SessionRequest {
     map<uint32, bytes> peerData = 3;
 }
 
+enum TerminatorPrecedence {
+    Default = 0;
+    Required = 1;
+    Failed = 2;
+}
+
 message CreateTerminatorRequest {
-    string id = 1;
     string serviceId = 2;
     string binding = 3;
     string address = 4;
     map<uint32, bytes> peerData = 5;
     uint32 cost = 6;
+    TerminatorPrecedence precedence = 7;
```

Comment:
I know "creating a terminator" is primarily to support the edge... why does the edge care about `TerminatorPrecedence`?

Reply:
In an HA scenario the primary will want to indicate that that it should be required, and the secondary will want to be default precedence.

Reply:
Because the edge needs to make that decision at Xgress dial time, right?

Reply:
Yes, the SDK will make a Bind call to the xgress_edge component and will include the cost and precedence. xgress_edge will that turn that into a create terminator call and pass along the cost and precedence.

Reply:
Makes sense. 👍